portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div class="row" id="header">
+            <div class="col-3" id="img">
+                <img src="images/cat_with_bread.jpg"></img>
+            </div>
+            <div class="col-3">
+            <h1>Hi, I am Masha Samsikova</h1>
+            <p class="annotation">Second-year CS student at <a href="https://en.itmo.ru/en/">ITMO University</a></p>
+            <p>And this is my <a href="https://github.com/holounic">github</a></p>
+          </div>
+        </div>
+        <br>
+        <h3> Implemented projects </h3>
```

Comment:
nit: Please remove the leading and trailing whitespace within the `<h3>` tag:
https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace